pep-0684.rst

```diff
@@ -1,13 +1,13 @@
 PEP: 684
 Title: A Per-Interpreter GIL
 Author: Eric Snow <ericsnowcurrently@gmail.com>
-Discussions-To: python-dev@python.org
+Discussions-To: `python-dev@python.org <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/>`__
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 08-Mar-2022
 Python-Version: 3.11
-Post-History: 08-Mar-2022
+Post-History: `08-Mar-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/`__
```

Comment:
```suggestion
Post-History: `08-Mar-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/>`__
```

Reply:
It doesn't show here, but it looks like this was already fixed while I was making my review.